script/oracles/config/ethereum.json

```diff
@@ -1,41 +1,28 @@
 [
-    {
-        "name": "WETH (wstETH)",
-        "collateralToken": "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0",
-        "loanToken": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-        "baseFeed1": "0x4F67e4d9BD67eFa28236013288737D39AeF48e79",
-        "baseFeed2": "0x0000000000000000000000000000000000000000",
-        "quoteFeed1": "0x0000000000000000000000000000000000000000",
-        "quoteFeed2": "0x0000000000000000000000000000000000000000",
-        "vaultConversionSample": 1,
-        "minPrice": 1100000000000000000000000000000000000,
-        "maxPrice": 1200000000000000000000000000000000000,
-        "salt": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-        "name": "USDC (wstETH)",
-        "collateralToken": "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0",
-        "loanToken": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
-        "baseFeed1": "0x4F67e4d9BD67eFa28236013288737D39AeF48e79",
-        "baseFeed2": "0x0000000000000000000000000000000000000000",
-        "quoteFeed1": "0x986b5E1e1755e3C2440e960477f25201B0a8bbD4",
-        "quoteFeed2": "0x0000000000000000000000000000000000000000",
-        "vaultConversionSample": 1,
-        "minPrice": 2000000000000000000000000000,
-        "maxPrice": 3500000000000000000000000000,
-        "salt": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-        "name": "USDC (wbIB01)",
-        "collateralToken": "0xABC56186c2Df63C338E04C9C9B9814E27983a5a9",
-        "loanToken": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
-        "baseFeed1": "0x32d1463EB53b73C095625719Afa544D5426354cB",
-        "baseFeed2": "0x0000000000000000000000000000000000000000",
-        "quoteFeed1": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
-        "quoteFeed2": "0x0000000000000000000000000000000000000000",
-        "vaultConversionSample": 1,
-        "minPrice": 100000000000000000000000000,
-        "maxPrice": 150000000000000000000000000,
-        "salt": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    }
+  {
+    "name": "USDC (WETH)",
```

Comment:
I thought we were going to use the ETH/USDC chainlink feed, according to [this message](https://morpholabs.slack.com/archives/C02N7CZ088N/p1704965952973829?thread_ts=1704960640.283929&cid=C02N7CZ088N). What changed ?

Reply:
@MerlinEgalite